package.json

```diff
@@ -42,31 +42,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "format": "prettier . --write",
+    "lint": "eslint src/"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
       "eslint:recommended",
       "plugin:react/recommended",
       "plugin:@typescript-eslint/recommended",
-      "plugin:jsx-a11y/strict"
+      "plugin:jsx-a11y/strict",
+      "plugin:prettier/recommended"
     ],
     "plugins": [
       "react-hooks",
       "@typescript-eslint",
       "jsx-a11y"
     ],
     "rules": {
+      "no-case-declarations": "off",
       "no-prototype-builtins": "off",
       "prefer-const": "warn",
+      "react/prop-types": "off",
```

Comment:
Similarly, depending on how many times this is happening, it may be worth trying to fix it. If there are too many instances, then please consider marking it as a warning instead.

Reply:
This one turned out to be unnecessary, I had meant to remove it; UoC have ignored specific instances of it